index.html

```diff
@@ -0,0 +1,164 @@
+            <a href="#" class="twitter"><i class="fa fa-twitter"></i></a>
+            <a href="#">My page</a>
+            <a href="#">Logout</a>
+          </div>
+        </nav>
+      </div>
+      <div class="row">
+        <nav class="navbar" id="navbar">
+          <div class="navbar-header">
+            <a class="navbar-brand" href="#">
+              <img src="./images/img4.png" alt="navbar logo imag" />
+            </a>
+          </div>
+          <ul class="nav navbar-nav navItem text-secondary" id="navMenu">
+            <li class="active"><a href="index.html">Home</a></li>
+            <li><a href="about.html">About</a></li>
+            <li><a href="#program">Program</a></li>
+            <li><a href="#sports">Sports</a></li>
+            <li><a href="#sponsers">Sponser</a></li>
+            <li><a href="#">Nutrition</a></li>
+          </ul>
+
+          <div class="humgurger">
+            <span class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
+          </div>
+        </nav>
+      </div>
+      <main class="main1">
+        <div class="row introduction">
+          <p>"Welcome to Healthline Nutrition!"</p>
+          <h1 class="secondHeader">
+            The Definitive Guide<br />
+            to Healthy Eating in Real Life
+          </h1>
+
+          <p id="second-p">
+            Healthy eating doesn’t have to be complicated. It’s entirely
+            possible to nourish your body while enjoying the foods you love.
+          </p>
+          <h3>2022.10.15(SAT) ~ 16(SUN)</h3>
+          <h5>@ Do not focus on how much you eat ,Focus on what you eat .</h5>
+        </div>
+        <div class="row mainProgram" id="program">
+          <h3>Main Program</h3>
+          <hr />
+          <div class="col-sm card">
+            <img src="./images/fruits.jpg" alt=" fruits" />
+            <h4 class="card-title">Fruits</h4>
+            <p class="card-text">
+              Fresh, frozen, or canned fruits are great choices. Try fruits
+              beyond apples and bananas such as mango, pineapple or kiwi fruit
+            </p>
+          </div>
+          <div class="col-sm card">
+            <img src="./images/vegtables.jpg" alt=" vegtables" />
+            <h4 class="card-title">Vegtables</h4>
+            <p class="card-text">
+              Fresh, frozen, or canned fruits are great choices. Try fruits
+              beyond apples and bananas such as mango, pineapple or kiwi fruit
+            </p>
+          </div>
+          <div class="col-sm card">
+            <img
+              src="./images/glass-clean-drinking-water-44066082.jpg"
+              alt=" Water"
+            />
+            <h4 class="card-title">Water</h4>
+            <p class="card-text">
+              Consuming 2 liters, or 8 glasses, of water a day, but it can also
+              come from dietary sources, such as fruit and vegetables.
+            </p>
```

Comment:
- [ ] Kindly fix this section to look more like in the template.
Your design 
![Screenshot from 2022-10-25 18-41-35](https://user-images.githubusercontent.com/89228743/197855824-7e222c0d-fa93-493f-804d-d833191b15e5.png)
The template
![Screenshot from 2022-10-25 18-44-07](https://user-images.githubusercontent.com/89228743/197856025-a57efffc-594b-4a62-ae39-3683055ed443.png)

Reply:
Bro, It is like the design but It is so much responsibility that when you continue to decrease the size of the screen this element increases its height to adjust.

Reply:
Hello @Anwaar12, thank you for the comment. I don't know then why it was not displaying properly at my end. But if you are sure that everything is working as expected you can submit for a another review.

Reply:
I have to change the style. I centered the element now it looks okay.